content/en/integrations/guide/service-account-impersonation.md

```diff
@@ -0,0 +1,111 @@
+2. Click on **Add GCP Account**. If you have no configured projects, you are automatically redirected to this page.
+3. If you have not generated a Datadog principal for your org, click the **Generate Principal** button.
+4. Copy your Datadog principal and keep it for the next section.
+   {{< img src="integrations/guide/service_account_impersonation/datadog-principal.png" alt="Datadog interface, showing the 'Add New GCP Account' flow. The first step, 'Add Datadog Principal to Google,' features a text box where a user can generate a Datadog Principal and copy it to their clipboard. The second step, 'Add Service Account Email,' features a text box that the user can complete in section 3." style="width:70%;">}}
+   Keep this window open for the [next section](#3-complete-the-integration-setup-in-datadog).
+5. In [Google Cloud console][9], under the **Service Acounts** menu, find the service account you created in the [first section](#1-create-your-google-cloud-service-account).
+6. Go to the **Permissions** tab and click on **Grant Access**.
+   {{< img src="integrations/guide/service_account_impersonation/grant-access.png" alt="Google Cloud console interface, showing the Permissions tab under Service Accounts." style="width:70%;">}}
+7. Paste your Datading principal into the **New principals** text box.
+8. Assign the role of **Service Account Token Creator** and click **Save**.
+   {{< img src="integrations/guide/service_account_impersonation/add-principals.png" alt="Google Cloud console interface, showing an 'Add principals' box and an 'Assign roles' interface." style="width:70%;">}}
+
+**Note**: If you previously configured access using a shared Datadog principal, you can revoke the permission for that principal after you complete these steps.
+
+### 3. Complete the integration setup in Datadog
+
+1. In your Google Cloud console, navigate to the **Service Account** > **Details** tab. There, you can find the email associated with this Google service account. It resembles `<sa-name>@datadog-sandbox.iam.gserviceaccount.com`.
+2. Copy this email.
+3. Return to the integration configuration tile in Datadog (where you coppied your Datadog principal in the [previous section](#2-add-the-datadog-principal-to-your-service-account)).
+4. In the box under **Add Service Account Email**, paste the email you previously coppied.
+5. Click on **Verify and Save Account**.
+
+In approximately fifteen minutes, metrics appear in Datadog.
+
+### 4. Assign roles to other projects (optional)
+
+Automatic project discovery simplifies to process of adding additional rpojects to be monitored. If you grant your service account access to other projects, folders, or orgs, Datadog discovers these projects (and any projects nested in the folders or orgs) and automatically adds them to your integration tile.
+
+1. Make sure you have the appropriate permissions to assign roles at the desired scope:
```

Comment:
Can we be consistent about the line spacing between the bullets please?

Reply:
I have been trying to troubleshoot this and legitimately cannot figure out why it looks like that